lms/templates/static_templates/404.html

```diff
@@ -10,11 +10,21 @@
 
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="outside-app">
-    <h1>${_("Page not found")}</h1>
-    <p>${Text(_('The page that you were looking for was not found. Go back to the {link_start}homepage{link_end} or let us know about any pages that may have been moved at {email}.')).format(
-        link_start=HTML('<a href="/">'),
-        link_end=HTML('</a>'),
-        email=HTML('<a href="mailto:{email}">{email}</a>').format(email=Text(static.get_tech_support_email_address()))
-      )}</p>
+        <h1>
+            <%block name="pageheader">${page_header or _("Page Not Found")}</%block>
```

Comment:
There are test failures, because the old code had "Page not found", and now we have "Page Not Found".